package.json

```diff
@@ -34,5 +34,8 @@
         "eslint-config-airbnb-base": "^12.1.0",
         "eslint-plugin-import": "^2.9.0",
         "jest": "^22.4.2"
+    },
+    "jest": {
+        "testURL": "http://localhost/"
```

Comment:
Add jest config in order to address failing unit tests. See https://github.com/jsdom/jsdom/issues/2304